theseus/labs/lie/lie_tensor.py

```diff
@@ -245,14 +245,13 @@ def _check_ltype(self, other: "LieTensor", op_name: str):
             raise ValueError("f{op_name} requires both tensors to have same ltype.")
 
     # Returns a new LieTensor with the given data and the same ltype as self
-    def new(  # type: ignore
+    def new_lietensor(  # type: ignore
         self, args: Any, device: Device = None, dtype: Optional[torch.dtype] = None
     ) -> "LieTensor":
         if isinstance(args, LieTensor):
             warnings.warn(
-                "Calling new() on a LieTensor results in shared data storage. "
                 "To copy construct from a LieTensor, it is recommended to use "
-                "lie_tensor.clone().",
+                "t.clone().detach() instead of lie_tensor.new_lietensor(t)",
```

Comment:
Assuming `t` is a `LieTensor`?

Reply:
Oops, that should be `"...lie_tensor.clone.detach()..."`. Good catch.